ktor-client/ktor-client-cio/jvm/test/io/ktor/client/engine/cio/CIOHttpsTest.kt

```diff
@@ -140,6 +141,7 @@ class CIOHttpsTest : TestWithKtor() {
     }
 
     @Test
+    @Ignore
```

Comment:
Why the test is ignored? Maybe leave a comment?

Reply:
Because of the availability of remote resources. I'm going to replace that test.